src/Etterna/Globals/NoteDataStructures.h

```diff
@@ -0,0 +1,39 @@
+// Only NoteData structures independent of the rest of Stepmania live here
```

Comment:
This entire file already exists at `/src/Etterna/Models/NoteData/NoteDataStructures.h`